natgeo/mortar
# [SYTO-869] Adds markup and rules needed to support different layout for the third and fourth card

https://jira.natgeo.com/browse/SYTO-869

To test:

- Pull SYTO-869
- Run `gulp test` to verify tests and linting pass
- Run `gulp serve`
- Verify component layout at http://localhost:3000/contentPackages/ or http://localhost:3000/contentPackages/contentPackage/index.html at the various break points
